test/Npgsql.PluginTests/GeoJSONTests.cs

```diff
@@ -13,8 +13,8 @@
 using static Npgsql.Tests.TestUtil;
 
 namespace Npgsql.PluginTests;
-
```

Comment:
Maybe add the comment to the solution file and comment the plugin test project to avoid duplicates